pyproject.toml

```diff
@@ -14,7 +14,7 @@ authors = ["Terra Quantum AG"]
 python = "^3.8"
 requests = "2.31.0"
 PyYAML = "^6.0.1"
-tq42_grpc_client = "1.0.120"
+tq42_grpc_client = "1.0.127"
```

Comment:
if you make any update to the sdk you also need to bump the version ;) so increment version with a patch in here.